common/utils/src/main/resources/error/error-classes.json

```diff
@@ -724,6 +724,11 @@
     ],
     "sqlState" : "42K04"
   },
+  "FAILED_RENAME_TEMP_FILE" : {
+    "message" : [
+      "Failed to rename temp file <srcPath> to <dstPath> as rename returned false."
```

Comment:
Which rename returned false. Could you make it clearer? Do you mean rename of filesystem.

Reply:
Change to `FileSystem.rename`.